### ISSUE TITLE
Maffucci type Enchondromatosis

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7109]
**Preferred term label:**

Maffucci type Enchondromatosis

**Synonyms**

**Definition (free text, please give PubMed ID)**

Text below from https://medlineplus.gov/genetics/condition/maffucci-syndrome/  which has citations at the bottom.

Indicates that these enchrondromatoses are specific for Maffucci syndrome.

Maffucci syndrome is a disorder that primarily affects the bones and skin. It is characterized by multiple enchondromas, which are noncancerous (benign) growths of cartilage that develop within the bones. These growths most commonly occur in the limb bones, especially in the bones of the hands and feet; however, they may also occur in the skull, ribs, and bones of the spine (vertebrae). Enchondromas may result in severe bone deformities, shortening of the limbs, and fractures.

The signs and symptoms of Maffucci syndrome may be detectable at birth, although they generally do not become apparent until around the age of 5. Enchondromas develop near the ends of bones, where normal growth occurs, and they frequently stop forming after affected individuals stop growing in early adulthood. As a result of the bone deformities associated with Maffucci syndrome, people with this disorder generally have short stature and underdeveloped muscles.

Maffucci syndrome is distinguished from a similar disorder that involves enchondromas (Ollier disease) by the presence of red or purplish growths in the skin consisting of tangles of abnormal blood vessels (hemangiomas). In addition to hemangiomas, individuals with Maffucci syndrome occasionally also have lymphangiomas, which are masses made up of the thin tubes that carry lymph fluid (lymphatic vessels). These growths may appear anywhere on the body.

**Parent term (use hpo.jax.org/app)**
HP:0030038 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Maffucci syndrome

**Your nano-attribution (ORCID)**
0000-0002-3302-4610

**Comment by pnrobinson** [comment]
@taylordm I think that this term is probably a Mondo term and not an HPO term. Please reopen if you disagree and we can work on a definition.

**Comment by nicolevasilevsky** [comment]
we have this in Mondo: MONDO_0013808 'Maffucci syndrome'

I will add the synonym to Mondo
